torch/nn/modules/transformer.py

```diff
@@ -181,8 +186,15 @@ class TransformerDecoder(Module):
         norm: the layer normalization component (optional).
 
     Examples::
-        >>> decoder_layer = nn.TransformerDecoderLayer(d_model, nhead)
-        >>> transformer_decoder = nn.TransformerDecoder(decoder_layer, num_layers)
+        >>> decoder_layer = nn.TransformerDecoderLayer(d_model=512, nhead=8)
+        >>> transformer_decoder = nn.TransformerDecoder(decoder_layer, num_layers=6)
+        >>> encoder_layer = nn.TransformerEncoderLayer(d_model=512, nhead=8)
+        >>> transformer_encoder = nn.TransformerEncoder(encoder_layer, num_layers=6)
+        >>> src = torch.rand(10,32,512)
+        >>> tgt = torch.rand(20,32,512)
+        >>> memory = transformer_encoder(src)
```

Comment:
You don't need to define a transform_encoder here. For demonstration purpose, you could send a random memory to transformer_decoder and it will be fine.
```
>>> memory = torch.rand(10,32,512)
>>> tgt = torch.rand(20,32,512)
>>> out = transformer_decoder(tgt, memory)
```